Mathport/Binary/ParseTLean.lean

```diff
@@ -248,16 +248,13 @@ def parseLine (line : String) : ParseM Unit := do
       | "#BC" => pure BinderInfo.instImplicit
       | s     => throw $ IO.userError s!"[parseBinderInfo] unexpected: {s}"
 
-
-def parseTLean (tlean : FilePath) : IO (Array EnvModification) := do
-  IO.FS.Handle.mk tlean IO.FS.Mode.read >>= fun h => do
-     let _ ← h.getLine -- discard imports
-     parseLines h |>.run' {}
-where
-  parseLines (h : IO.FS.Handle) : ParseM (Array EnvModification) := do
-    while (not (← h.isEof)) do
```

Comment:
Should we deprecate `isEof`? It looks like an unnecessary footgun compared to "end of file is marked by returning no data".

Reply:
> "end of file is marked by returning no data *once*"

I'm not sure what's the bigger footgun here..  Either way, we should explain this is in a docstring somewhere.